lib/tasks_private/scvmm.rake

```diff
@@ -0,0 +1,54 @@
+require 'winrm'
+require 'json'
+require 'yaml'
+
+namespace :manageiq do
+  namespace :providers do
+    namespace :scvmm do
+      desc "Regenerate VCR cassette"
+      task :regenerate_cassette, [:username, :password, :host, :port] do |_t, args|
+        secrets = Rails.application.secrets.scvmm
```

Comment:
They come from `spec/manageiq/config/secrets.yml`. The last time I tried it under `<gem_root>/config/secrets.yml` it didn't work.